src/Microsoft.Identity.Web.TokenAcquisition/AspNetCore/MicrosoftIdentityAppCallingWebApiAuthenticationBuilder.cs

```diff
@@ -49,7 +50,7 @@ public class MicrosoftIdentityAppCallsWebApiAuthenticationBuilder : MicrosoftIde
             }
 
             Services.AddHttpContextAccessor();
-            Services.AddSingleton<IMsalTokenCacheProvider, MsalMemoryTokenCacheProvider>();
+            Services.TryAddSingleton<IMsalTokenCacheProvider, MsalMemoryTokenCacheProvider>();
```

Comment:
We might want to change the XML comment to say "Add in memory token caches, if no other implementation of IMsalTokenCacheProvider was already added."